File: sorting/selectionSort/selection.c
#include<stdio.h>
void selection(int *arr,int n){
    for(int i=0;i<n;i++){
        int min=i;
        for(int j=i;j<n;j++){
            if(arr[j]<arr[min]){
                min=j;
            }
        }
        int temp=arr[min];
        arr[min]=arr[i];
        arr[i]=temp;
    }
}
int main(){
    int arr[5]={5,3,1,2,4};
    selection(arr,5);
    for(int i=0;i<5;i++){
        printf("%d ",arr[i]);
    }
}

File: sorting/selectionSort/selection.py
def selection(l):
    for i in range(0,len(l)):
        min=i
        for j in range(i,len(l)):
            if(l[j]<l[min]):
                min=j
        l[min],l[i]=l[i],l[min]
    return l
l=[5,3,1,2,4]
l=selection(l)
print(l)
